src/search-bar/search.jsx

```diff
@@ -0,0 +1,25 @@
+import './search.css'
+import searchlogo from '../assets/search.svg'
+import React, { useEffect } from 'react';
+
+
+
+const Search = () => {
```

Comment:
to use this component I think we will have to pass a `value` and `onChange` props to store the input value and handle user typing on this component.